avm/utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -978,7 +1110,79 @@ Describe 'Test file tests' -Tag 'TestTemplate' {
+
+      param(
+        [object[]] $testFileContent
+      )
+
+      if (($testFileContent | Out-String) -match "param serviceShort string = '(.*)'") {
+        $Matches[1] | Should -BeLike "*min"
+      } else {
+        Set-ItResult -Skipped -Because 'the module test deployment file should contain a parameter [serviceShort] using the syntax [param serviceShort string = ''*min''] but it doesn''t.'
+      }
+    }
+
+    It "[<moduleFolderName>] [<testName>] Bicep test deployment files in a [max] folder should have a [serviceShort] parameter with a value ending with  [max]" -TestCases ($deploymentTestFileTestCases | Where-Object { $_.testFilePath -match '.*[\\|\/]max[\\|\/].*' }) {
+
+      param(
+        [object[]] $testFileContent
+      )
+
+      if (($testFileContent | Out-String) -match "param serviceShort string = '(.*)'") {
+        $Matches[1] | Should -BeLike "*max"
+      } else {
+        Set-ItResult -Skipped -Because 'the module test deployment file should contain a parameter [serviceShort] using the syntax [param serviceShort string = ''*max''] but it doesn''t.'
+      }
+    }
+
+    It "[<moduleFolderName>] [<testName>] Bicep test deployment files in a [waf-aligned] folder should have a [serviceShort] parameter with a value ending with [waf]" -TestCases ($deploymentTestFileTestCases | Where-Object { $_.testFilePath -match '.*[\\|\/]waf\-aligned[\\|\/].*' }) {
+
+      param(
+        [object[]] $testFileContent
+      )
+
+      if (($testFileContent | Out-String) -match "param serviceShort string = '(.*)'") {
+        $Matches[1] | Should -BeLike "*waf"
+      } else {
+        Set-ItResult -Skipped -Because 'the module test deployment file should contain a parameter [serviceShort] using the syntax [param serviceShort string = ''*waf''] but it doesn''t.'
+      }
+    }
+
+    It "[<moduleFolderName>] [<testName>] Bicep test deployment files should contain a metadata string [name]" -TestCases $deploymentTestFileTestCases {
+
+      param(
+        [object[]] $testFileContent
+      )
+      ($testFileContent | Out-String) | Should -Match "metadata name = .+" -Because 'Test cases should contain a metadata string [name] in the format `metadata name = ''This is one hell of a test name''` to be more descriptive. If provided, the tooling will automatically inject it into the module''s readme.md file.'
+    }
+
+    It "[<moduleFolderName>] [<testName>] Bicep test deployment files should contain a metadata string [description]" -TestCases $deploymentTestFileTestCases {
+
+      param(
+        [object[]] $testFileContent
+      )
+      ($testFileContent | Out-String) | Should -Match "metadata description = .+" -Because 'Test cases should contain a metadata string [description] in the format `metadata description = ''This is one hell of a description''` to be more descriptive. If provided, the tooling will automatically inject it into the module''s readme.md file.'
```

Comment:
Same here btw :fire:

Reply:
I know. Hence I updated it in both locations directly